.github/new-prs-labeler.yml

```diff
@@ -243,6 +243,50 @@ tools:llvm-mca:
   - llvm/include/llvm/MCA/**/*
   - llvm/lib/MCA/**/*
 
+openmp:clang:
+  - clang/include/clang/Basic/OpenMP*
+  - clang/include/clang/AST/OpenMPClause.h
+  - clang/include/clang/AST/DeclOpenMP.h
+  - clang/include/clang/AST/ExprOpenMP.h
+  - clang/include/clang/AST/StmtOpenMP.h
+  - clang/lib/AST/DeclOpenMP.cpp
+  - clang/lib/AST/OpenMPClause.cpp
+  - clang/lib/AST/StmtOpenMP.cpp
+  - clang/lib/Headers/openmp_wrappers/*
+  - clang/lib/Parse/ParseOpenMP.cpp
+  - clang/lib/Basic/OpenMPKinds.cpp
+  - clang/lib/Driver/ToolChains/AMDGPUOpenMP.cpp
+  - clang/lib/Driver/ToolChains/AMDGPUOpenMP.h
+  - clang/lib/CodeGen/CgStmtOpenMP.cpp
+  - clang/lib/CodeGen/CGOpenMP*
+  - clang/lib/Sema/SemaOpenMP.cpp
+  - clang/test/OpenMP/*
+  - clang/test/AST/ast-dump-openmp-*
+  - llvm/lib/Frontend/OpenMP/*
+  - llvm/lib/Transforms/IPO/OpenMPOpt.cpp
+  - llvm/include/llvm/Frontend/OpenMP/*
+  - llvm/include/llvm/Transforms/IPO/OpenMPOpt.h
+  - llvm/unittests/Frontend/OpenMP*
+  - llvm/test/Transforms/OpenMP/*
+
+openmp:flang:
+  - flang/test/**/OpenMP/*
+  - flang/lib/Lower/OpenMP.cpp
+  - flang/lib/Semantics/resolve-directives.cpp
+  - flang/lib/Semantics/check-omp-structure.cpp
+  - flang/lib/Optimizer/Transforms/OMP*
+  - flang/test/Fir/convert-to-llvm-openmp-and-fir.fir
+  - flang/test/Lower/OpenMP/*
+  - flang/test/Transforms/omp*
+  - mlir/**/OpenMP/*
+  - mlir/test/Target/LLVMIR/openmp*
+
+openmp:libomp:
+  - any: ['openmp/**', '!openmp/libomptarget/*']
```

Comment:
Shouldn't that be `all`? ie, that might always be true

Reply:
I tested this in https://github.com/jplehr/llvm-project/pulls and it appeared to be working as expected.

Reply:
I carefully re-red the documentation, and I believe that the difference is that the `any` rule applies to any path modified in the PR, whereas the `all` rule applies to all paths in the PR.
In this case, we want to label the change with OpenMP as soon as it touches something in these paths, even though it may also touch other paths.